services/masterdata/src/lib/components/masterdatas-details/masterdatas-details.component.ts

```diff
@@ -50,11 +52,9 @@ export class MasterdatasDetailsComponent implements OnInit {
 	}
 
 	prepareForDisplay(item: Masterdata) {
-		return Object.entries(item).map(item => {
-			item[0] = item[0].toLowerCase().replace("masterdata", "");
-			if (["typeid", "typename"].includes(item[0])) {
-				item[0] = `type ${item[0].replace("type", "")}`;
-			}
+		// "metadata" model field will have a special display!
+		return Object.entries(item).filter(item => item[0] !== "metadata").map(item => {
+			item[0] = this.translateService.instant(`masterdatamgmt.pages.masterdataDetails.modelFields.${item[0]}`);
 			return item;
```

Comment:
unreachable code due to duplicate return statement. Second time I see this code. Should this be moved to a separate file?

Reply:
@Johanh-od there are 2 private methods in separate classes (one for Masterdata Type, the other for Masterdatas), which treat the case for metadata, but with different model and language translation variable. I suppose I can make a generic method for this in a file, something like services/masterdata/src/lib/core/utils/masterdata.functions.ts?